airlock_processor/shared_code/blob_operations.py

```diff
@@ -29,25 +42,21 @@ def copy_data(source_account_name: str, destination_account_name: str, request_i
                                                    credential=credential)
     source_container_client = source_blob_service_client.get_container_client(container_name)
 
-    try:
```

Comment:
I've removed this try-catch block as the request ID will still be logged on lines 51 and 57